### PR TITLE
[RELEASE-1.8] Re-introducing https://github.com/knative/serving/pull/13337

### PR DESCRIPTION
**Pull request by ReToCode**
JIRA: https://issues.redhat.com/browse/SRVCOM-2300

**Changes**
- Re-introducing upstream change: https://github.com/openshift-knative/serving/commit/42d13773f35e4230fdf1cebe8aa6f08720c5cf9d

/hold https://github.com/openshift/release/pull/36606 is necessary for the tests to pass.
/cc @mgencur, @skonto

### CODE REVIEW

**Comment by nak3** [comment]
It seems 1.8 branch missed https://github.com/knative/serving/commit/a6a18b857be4f9e03a5bc4e196ea8450ff68828e but we need to cherry-pick?

File: config/core/deployments/activator-hpa.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: autoscaling/v2beta2
+apiVersion: autoscaling/v2
```
Comment (nak3) [inline config/core/deployments/activator-hpa.yaml:15]: We need to bump manifests for CI. Could you please run:

```
 ./openshift/release/generate-release.sh knative-v1.8.0
 ./openshift/release/generate-release.sh ci
```

? These will bump `openshift/release/knative-serving-*` files. 

Also, we need to update the manifests in `openshift/release/artifacts/*` for SO repo.

**Comment by ReToCode** [comment]
> It seems 1.8 branch missed https://github.com/knative/serving/commit/a6a18b857be4f9e03a5bc4e196ea8450ff68828e but we need to cherry-pick?

I think that is already handled in the SO. It will change the versions depending on the OCP version. If we merge this, we might break it for OCP 4.10.

**Comment by nak3** [comment]
> It seems 1.8 branch missed https://github.com/knative/serving/commit/a6a18b857be4f9e03a5bc4e196ea8450ff68828e but we need to cherry-pick?

And if we cherry-pick it we need to add a patch in `openshift/release/artifacts/2-serving-core.yaml` as well. Sorry for bothering you..

**Comment by ReToCode** [comment]
> And if we cherry-pick it we need to add a patch in openshift/release/artifacts/2-serving-core.yaml as well. Sorry for bothering you..

Hm but then let's not cherry-pick it. S-O does handle that transformation anyways. I think this is why it was not on the revert-list in the first place as it was introduced upstream in 1.9. WDYT? /cc @skonto

**Comment by nak3** [comment]
Sure, no problem for me to handle it in SO's transformation if it is more simple than the temporary patch here.

**Pull request merged**